packages/zosjobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Added waitForOutput & waitForActive as optional parameters to download on zosjobs. [#2326] (https://github.com/zowe/zowe-cli/pull/2326).
+- Enhancement: Added --wait-for-output & --wait-for-active as optional parameters to download on zosjobs.
```

Comment:
I guess we don't need this entry here since it's already in the CLI/changelog.md 😋 
```suggestion
```